src/pages/cookie-policy/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import styles from 'styled-components';
+
+/**
+ * Cookie Policy
+ **/
+
+const CookiePolicyContainer = styles.div`
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+  margin-top: 20px;
```

Comment:
remove `margin-top: 20px;` ,  `margin-bottom: 20px;` and update `margin: 20px auto;`